src/promptflow/setup.py

```diff
@@ -50,6 +50,8 @@
     "filetype>=1.2.0",  # used to detect the mime type for mulitmedia input
     "jsonschema>=4.0.0,<5.0.0",  # used to validate tool
     "docutils",  # used to generate description for tools
+    "uvicorn>=0.25.0", # used to start local executor service
```

Comment:
which scenario will this server be used?

Reply:
For portal, we will separate the runtime and executor into two different containers. This is just a draft PR, not sure what to use in the end.